utils_test.go

```diff
@@ -0,0 +1,99 @@
+package ocp_dnsnameresolver
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestIsWildcard(t *testing.T) {
+	tests := []struct {
+		dnsName        string
+		expectedOutput bool
+	}{
+		// success
+		{"*.example.com", true},
+		{"*.sub1.example.com", true},
+		// negative
+		{"www.example.com", false},
+		{"sub2.sub1.example.com", false},
+	}
+
+	for _, test := range tests {
+		actualOutput := isWildcard(test.dnsName)
+		require.Equal(t, actualOutput, test.expectedOutput)
```

Comment:
Is it worth bringing `require` dependency? `TestIsSameNextLookupTime` worked out well without.

Reply:
I have removed the dependency on `require` package for the tests.